releases/4.0.2.md

```diff
@@ -0,0 +1,218 @@
+    - Fix the infinite follower/learner retry issue when `tidb_replica_read` is set to `follower` and there is a network partition between the leader and follower/learner. [#17443](https://github.com/pingcap/tidb/pull/17443)
+    - Fix the issue that TiDB sends too much pings to PD follower occasionally. [#17947](https://github.com/pingcap/tidb/pull/17947)
+    - Fix the issue that old versioned range partition table can not be loaded to TiDB 4.0.0. [#17983](https://github.com/pingcap/tidb/pull/17983)
+    - Fix the SQL command timeout issue when multiple region requests failed at the same time by assigning different `Backoffer` for each region. [#17585](https://github.com/pingcap/tidb/pull/17585)
+    - Fix the incompatible behavior on parsing datetime delimiters. [#17501](https://github.com/pingcap/tidb/pull/17501)
+    - Fix the issue that TiKV request occasionally send to TiFlash server. [#18105](https://github.com/pingcap/tidb/pull/18105)
+
++ TiKV
+    - Fix a memory safety problem for status server. [#8101](https://github.com/tikv/tikv/pull/8101)
+    - Fix precision lost in json numeric comparsion [#8087](https://github.com/tikv/tikv/pull/8087)
+    - Fix wrong query slow log [#8050](https://github.com/tikv/tikv/pull/8050)
+    - Fix a case that a peer can not be removed when its store is isolated during multiple merge process. [#8048](https://github.com/tikv/tikv/pull/8048)
+    - Fix `tikv-ctl recover-mvcc`  doesn't remove invalid pessimistic locks. [#8047](https://github.com/tikv/tikv/pull/8047)
+    - Fix missing titan histogram metrics [#7997](https://github.com/tikv/tikv/pull/7997)
+    - Fix a problem that TiKV returns `duplicated error` to TiCDC. [#7887](https://github.com/tikv/tikv/pull/7887)
+    - Improve PD client panic log. [#8093](https://github.com/tikv/tikv/pull/8093)
+    - Add back `process_cpu_seconds_total` and `process_start_time_seconds`. [#8029](https://github.com/tikv/tikv/pull/8029)
+
++ PD
+    - Add config check for `pd-server.dashboard-address` of PD config. [#2517](https://github.com/pingcap/pd/pull/2517)
+    - Fix the issue that PD may panic when setting `store-limit-mode` to `auto` [#2544](https://github.com/pingcap/pd/pull/2544)
+    - Fix the issue that hotspot cannot be identified in some cases [#2463](https://github.com/pingcap/pd/pull/2463)
+    - Fix the placement rules issue that prevent store changing to tombstone [#2546](https://github.com/pingcap/pd/pull/2546)
+    - Fix the issue that PD may panic when upgrading from the previous version [#2564](https://github.com/pingcap/pd/pull/2564)
+
++ TiFlash
+    - Fix the issue that proxy may panic when meet region not found [#807](https://github.com/pingcap/tics/pull/807)
+    - Fix the issue that I/O exception thrown in drop table may lead to TiFlash schema sync failure. [#791](https://github.com/pingcap/tics/pull/791)
+
++ Tools
+    - TiDB Binlog
+        -
+    - TiCDC
+        -
+    - Lignting
+        -
+
+## TiKV
```

Comment:
@uglyengineer Are these PRs (Line 174 to Line 218) included in this release notes, or should they be removed?

Reply:
> @uglyengineer Are these PRs (Line 174 to Line 218) included in this release notes, or should they be removed?

@TomShawn please remove; they are already included above.